src/index.js

```diff
@@ -1,112 +1,102 @@
-import readPkgUp from 'read-pkg-up';
-import HtmlWebpackIncludeAssetsPlugin from 'html-webpack-include-assets-plugin';
-import ExternalModule from 'webpack/lib/ExternalModule';
-import resolvePkg from 'resolve-pkg';
+const readPkgUp = require('read-pkg-up');
+const ExternalModule = require('webpack/lib/ExternalModule');
+const resolvePkg = require('resolve-pkg');
 
-import getResolver from './get-resolver';
-
-const pluginName = 'dynamic-cdn-webpack-plugin';
-let HtmlWebpackPlugin;
+let HtmlWebpackPlugin = null;
 try {
-    // eslint-disable-next-line import/no-extraneous-dependencies
     HtmlWebpackPlugin = require('html-webpack-plugin');
-} catch (err) {
-    HtmlWebpackPlugin = null;
-}
+// eslint-disable-next-line unicorn/prefer-optional-catch-binding
+} catch (_) {}
```

Comment:
This doesn't improve legibility and I don't see any improvement in performance either.